qCC/ccHistogramWindow.cpp

```diff
@@ -91,6 +97,11 @@ ccHistogramWindow::~ccHistogramWindow()
 	clearInternal();
 }
 
+void ccHistogramWindow::setSelectableItemControls(SHOW_SELECTABLE_ITEMS showItems)
```

Comment:
So this is the part which is duplicate with Andy's PR? Not sure how you'll handle that

Reply:
I am open going with his, I'm not overly attached to the way I proposed. I figured I'd wait for his response before modifying it again though :)

Reply:
After discussion in Slack with Chris, I merged my changes and he will adapt his PR.